src/platform/shared/itf/content.js

```diff
@@ -0,0 +1,165 @@
+  <va-loading-indicator set-focus message={message} label={label} />
+);
+
+export const ItfSearchSpinner = () =>
+  showLoading(
+    'Please wait while we check to see if you have an existing Intent to File.',
+    'looking for an intent to file',
+  );
+
+export const ItfCreateSpinner = () =>
+  showLoading(
+    'Please wait while we create your Intent to File.',
+    'creating an intent to file',
+  );
+
+export const itfMessage = ({ headline, content, status }) => (
+  <va-alert visible status={status}>
+    <h2 slot="headline">{headline}</h2>
+    {content}
+  </va-alert>
+);
+
+export const ItfCreatedAlert = ({ itfType, expirationDateFormatted }) =>
+  itfMessage({
+    status: 'success',
+    headline: `Your intent to file a ${itfType} application`,
+    content: (
+      <>
+        <div>
+          We automatically recorded your intent to file. If you’re approved,
+          we’ll use your intent to file date as your potential start date for
+          potential benefits.
+        </div>
+        <p>
+          <span>
+            Submit your application by{' '}
+            <strong>{expirationDateFormatted}</strong>. If you’re not ready to
+            submit your application by then, don’t worry. You can start a new
+            application, and we’ll record a new benefits start date.
+          </span>
+        </p>
+      </>
+    ),
+  });
+
+ItfCreatedAlert.propTypes = {
+  expirationDateFormatted: PropTypes.string.isRequired,
+  itfType: PropTypes.string.isRequired,
+};
+
+export const ItfFoundAlert = ({ itfType, expirationDateFormatted }) =>
+  itfMessage({
+    status: 'success',
+    headline: 'We’ve recorded your intent to file',
+    content: (
+      <>
+        <div>
+          {`We’ve found your intent to file a ${itfType} application in our records.`}
+        </div>
+        <p>
+          <span>
+            Submit your application by{' '}
+            <strong>{expirationDateFormatted}</strong>. If you don’t submit your
+            application by that date and you still want to apply, you can start
+            a new application. And, we’ll set a new benefits start date.
+          </span>
+        </p>
+        <va-link
+          external
+          href="/resources/your-intent-to-file-a-va-claim/"
+          text="Learn more about intent to file"
+        />
+      </>
+    ),
+  });
+
+ItfFoundAlert.propTypes = {
+  expirationDateFormatted: PropTypes.string.isRequired,
+  itfType: PropTypes.string.isRequired,
+};
+
+export const ItfFailedAlert = ({ itfType }) =>
+  itfMessage({
+    status: 'info',
+    headline: `You may want to confirm your intent to file for ${itfType} benefits`,
+    content: (
+      <>
+        <div>
+          We tried to check for your intent to file in our records, but there
+          was a problem with our system. You can still able to work on your
+          application.
+        </div>
+        <p>
+          Submitting your intent to file a Veterans pension application is
```

Comment:
If this is a shared component, should the content of this be more dynamic and/or not specify a specific team?

Reply:
You can pass in custom found, created and error alerts into the component.

Reply:
and yeah that should be more generic... I'll work on these typos.

Reply:
Fixed